lib/find.ts

```diff
@@ -31,7 +32,9 @@ export async function find(
     fileResults.push(path);
   };
 
-  await traverse(src, handler, (p) => excluded.includes(p));
+  await traverse(src, handler, (path) => {
+    return !!excludePatterns.find((pattern) => minimatch(path, pattern));
```

Comment:
yea maybe. In our case it's because it provide us with the small subset that we only need.
(If we used the snyk fast-glob it would touch the filesystem which we already have code to do, that does it pretty efficient. Otherwise we need I/O calls times two)